src/styles/components.css

```diff
@@ -67,16 +67,7 @@
   }
 
   .checkbox {
-    @apply appearance-none relative w-4 h-4 shrink-0 cursor-pointer disabled:cursor-default border border-gray-d2 dark:border-white rounded focus:outline-none focus:ring-2 focus:ring-blue-d1 disabled:bg-gray-l5 ring-offset-1 disabled:dark:bg-navy;
-  }
-  .checkbox:checked::before {
-    content: "✓";
-  }
-  .checkbox::before {
-    @apply text-sm absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2;
-  }
-  .checkbox[aria-invalid="true"] {
-    @apply border-red;
+    @apply accent-blue-d1 size-4 shrink-0 focus:outline-none focus:ring-2 focus:ring-blue-d1 disabled:accent-gray-l5 ring-offset-1;
```

Comment:
no need for elaborate custom checkbox look
<img width="103" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/d7139791-a3e2-4f2a-8ce0-df1b92093ed5">

where native look + [accent](https://tailwindcss.com/docs/accent-color#setting-the-accent-color)`accent-blue-d1` is sufficient
<img width="190" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/babf6a08-8f9e-4c2d-8ee0-dcc2806ffd3c">